sw/device/tests/BUILD

```diff
@@ -119,6 +113,31 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "aes_interrupt_encryption_test",
+    srcs = ["aes_interrupt_encryption_test.c"],
+    exec_env = dicts.add(
+        {
```

Comment:
I think that you want to add `EARLGREY_SILICON_OWNER_ROM_EXT_ENVS`.

Reply:
I see this is now done with the new BUILD file.